core/src/main/scala/kafka/server/BrokerServer.scala

```diff
@@ -433,6 +433,7 @@ class BrokerServer(
         config.shareGroupConfig.shareGroupDeliveryCountLimit,
         config.shareGroupConfig.shareGroupPartitionMaxRecordLocks,
         config.shareGroupConfig.shareFetchPurgatoryPurgeIntervalRequests,
+        config.shareGroupConfig.shareFetchMaxFetchRecords(),
```

Comment:
nit: Just being consistent with the above values, we can get rid of the parenthesis.

Reply:
Thanks for pointing, my bad. Done.